### DISCUSSION TITLE
Having 'emojis/cross' instead of a 'check/cross' to mark off habits

### DISCUSSION THREAD

**Discussion by snigam3112** [iSoron/uhabits#511] [Feature Requests]
For example, when I check off the habit for "brush nightly", instead of getting a check mark, I could get a tooth-emoji (🦷) show up instead. I think this makes the app more visual without requiring too many changes. In this way, it would be similar to chains.cc

I should be able to choose what emoji represents success when I create the habit and that is basically what is used to replace all the 'check' marks instead.

**Comment by iSoron** [comment]
Thank you for the feedback, @snigam3112. That's an interesting idea. Not sure how well would it work, but if someone wants to build a prototype, please feel free.
